lib/galaxy/model/__init__.py

```diff
@@ -4544,12 +4550,12 @@ def to_dict(self, view='collection', value_mapper=None):
         return rval
 
 
-class WorkflowInvocationStepJobAssociation(Dictifiable):
+class WorkflowInvocationStepJobAssociation(Dictifiable, RepresentById):
```

Comment:
I just found out this class can be removed, it was introduced in commit d5f98f49045bca351e982f17cac9386308089eeb , but its use was removed in commit e82012bde154c456f24743f3b61c0ed9ffc7eca2 , both part of PR #5013.